CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Expo SDK minimum version is 49 ([#3453](https://github.com/getsentry/sentry-react-native/pull/3453))
```

Comment:
This is gonna be a breakchange?

Reply:
No, we never wanted to support older than 49 and the only release this was part of is `5.16.0-alpha.1`.